From c34230e22d6a4e3e82db7e68db9764ed6a5fc5ff Mon Sep 17 00:00:00 2001
From: Thomas Preud'homme <robotux@celest.fr>
Date: Fri, 6 Jan 2012 18:14:34 +0100
Subject: Compile tccasm.c conditionally (TCC_CONFIG_ASM)

Only compile the content of tccasm.c if inline assembly is supported for
this architecture by testing the presence of macro TCC_CONFIG_ASM.

Origin: vendor
Bug-Debian: http://bugs.debian.org/654904
Forwarded: http://repo.or.cz/w/tinycc.git/commit/519a9040a16f86a7b309accf1cb54ed64d2f921f
Last-Update: 2012-03-05
Applied-Upstream: commit: 519a9040a16f86a7b309accf1cb54ed64d2f921f
---
 tccasm.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/tccasm.c b/tccasm.c
index 527c006..d1d2af1 100644
--- a/tccasm.c
+++ b/tccasm.c
@@ -19,6 +19,7 @@
  */
 
 #include "tcc.h"
+#ifdef CONFIG_TCC_ASM
 
 ST_FUNC int asm_get_local_label_name(TCCState *s1, unsigned int n)
 {
@@ -1115,3 +1116,4 @@ ST_FUNC void asm_global_instr(void)
 
     cstr_free(&astr);
 }
+#endif /* CONFIG_TCC_ASM */
